tests/QueryTest.php

```diff
@@ -181,31 +183,31 @@ public function testWhereDate(): void
 
     public function testWhereDay(): void
     {
-        $day = Birthday::whereDay('day', '12')->get();
+        $day = Birthday::whereDay('birthday', 12)->get();
         $this->assertCount(4, $day);
 
-        $day = Birthday::whereDay('day', '11')->get();
+        $day = Birthday::whereDay('birthday', 11)->get();
         $this->assertCount(1, $day);
     }
 
     public function testWhereMonth(): void
     {
-        $month = Birthday::whereMonth('month', '04')->get();
+        $month = Birthday::whereMonth('birthday', 4)->get();
         $this->assertCount(1, $month);
 
-        $month = Birthday::whereMonth('month', '05')->get();
+        $month = Birthday::whereMonth('birthday', 5)->get();
         $this->assertCount(5, $month);
     }
 
     public function testWhereYear(): void
     {
-        $year = Birthday::whereYear('year', '2021')->get();
+        $year = Birthday::whereYear('birthday', 2021)->get();
```

Comment:
Don't modify current tests to see if it's breaking current applications.

Reply:
as for this, it is breaking change.
Why?
its from comparing between string (current whereYear etc) to comparing datetime (on mongodb) with string/int
So what do i need to do?

Reply:
Hello,

Just don't touch current tests and add additional to see if it changes in behavior.

Thanks!

Reply:
@Davpyu can you revert old code and additional test cases ?

Reply:
Hello, i've reverted this. Please let me know if there are any commit that need to be reverted